time_execution/backends/elasticsearch.py

```diff
@@ -89,9 +99,12 @@ def write(self, name, **data):
         data["name"] = name
         data["timestamp"] = datetime.utcnow()
 
-        self.client.index(
-            index=self.get_index(),
-            doc_type=self.doc_type,
-            id=None,
-            body=data
-        )
+        try:
```

Comment:
The client keeps a pool of connections to elasticsearch, this should minimize the overhead.